ee_tests/src/interactions/launcher_interactions.ts

```diff
@@ -66,15 +66,10 @@ class LauncherInteractionsImpl implements LauncherInteractions {
         await this.selectPipelineStep(pipeline);
 
         logger.info('Check import application summary');
-        let summaryPage = new SummaryPage();
-        await summaryPage.clickImport();
+        await this.summaryImportPageSetup(name, repository, pipeline);
 
         logger.info('Check import application results');
-        let resultPage = new ResultsPage();
-
-        await screenshotManager.save('launcher');
-
-        await resultPage.clickReturnToDashboard();
+        this.resultPageStep(5);
```

Comment:
You miss `await` keyword here. If you run tests and check the screenshot you can see that the launcher is not finished at the end of the test.